regions_www/m/server.R

```diff
@@ -41,6 +41,9 @@ if (!on_server) {
 
 repo_sha <-  as.character(readLines(file.path(interface_root, "repo_sha")))
 
+# Check if the branch is production or not
+is_prod <- ifelse(system("git symbolic-ref -q --short HEAD", intern = T) == "production", TRUE, FALSE)
```

Comment:
The code isn't inside a git repo in production so sadly this doesn't work:

```bash
root@npct0:/var/shiny/pct-shiny# git symbolic-ref -q --short HEAD
fatal: Not a git repository (or any of the parent directories): .git
```
Which I think you spotted first https://github.com/npct/pct-shiny/commit/76cd62e40e103e29f64d946f6f96bb10aa5ea710

I believe we are after
```R
# Check if we are on the production server (npt followed by any number of digits (only) is a prod machine)
production_branch <- grepl("npt\\d*$", Sys.info()["nodename"])
```
(from deb05c6)

Reply:
Very good point @nikolai-b  - thanks!
Will make the required changes.  So glad you checked it before it got merged.
I had a recollection that we did something like this before, but couldn't find the commit - may need to learn more about git, it seems.

Reply:
Sure!

I used [git log -S](https://git-scm.com/docs/git-log#Documentation/git-log.txt--Sltstringgt) with `git log -S production` to list relevant commits then I just read the commit messages and used `git show`

Reply:
Noted, thanks!